python/examples/workflows/emitter.py

```diff
@@ -92,4 +92,5 @@ def post_process(state: State) -> WorkflowReservedStepName:
     try:
         asyncio.run(main())
     except FrameworkError as e:
+        traceback.print_exc()
```

Comment:
Here too: If you could remove the try except block from main above, it's no longer needed with this